File: CAPITULO 01/C++/ex5.cpp
#include <stdio.h>
int main ()
{
	double sal_atual,perc,sal_novo;
	printf("Digite o salario atual do funcionario :");
	scanf("%d%*c",&sal_atual);
	sal_novo = sal_atual+(sal_atual*(perc/100));
	printf("Novo salario no valor de %3.2d",sal_novo);
	getchar();
	return 0;
}

File: CAPITULO 01/Java/ex5.java
import java.io.*;
import java.util.*;
import java.text.*;
public class ex5{
	public static void main(String[]args){
		double sal_atual,percent,sal_novo;
		Scanner dado;
		dado = new Scanner (System.in);
		System.out.println("Informe o valor do salario do funcionario :");
		sal_atual = dado.nextDouble();
		System.out.println("Digite o percentual de aumento ");
		percent = dado.nextDouble();
		sal_novo = sal_atual + (sal_atual*(percent/100));
		DecimalFormat cs;
		cs = new DecimalFormat("0.00");
		System.out.println("Valor do salario atualizado e de "+cs.format(sal_novo));
		
	}
}